packages/core/src/application.ts

```diff
@@ -30,6 +30,12 @@ export class Application extends Context {
         this.server(options.servers[name], name);
       }
     }
+
+    if (options.controllers) {
+      for (const controller in options.controllers) {
+        this.controller(controller);
```

Comment:
This is wrong. It should be `this.controller(options.controllers[controller])`. The current code fails CI as follows:
```
packages/core/src/application.ts(36,25): error TS2345: Argument of type 'string' is not assignable to parameter of type 'Constructor<any>'.
```

Reply:
If `options.controllers` is an array, `for...in` loop should not be used. Use 'for...of` instead.